install/project/includes/configuration/configuration.inc.sample.php

```diff
@@ -165,9 +166,11 @@
 			define ('__APP_CSS_ASSETS__', __PROJECT_ASSETS__ . '/css');
 			define ('__APP_IMAGE_ASSETS__', __PROJECT_ASSETS__ . '/images');
 			define ('__APP_PHP_ASSETS__', __PROJECT_ASSETS__ . '/php');
+			define ('__APP_IMAGE_CACHE__', __APP_IMAGE_ASSETS__ . '/cache');
+			define ('__APP_UPLOAD__', __DOCROOT__ . __PROJECT_ASSETS__ . '/upload');
 			
 			define ('__PLUGIN_ASSETS__',  __SUBDIRECTORY__ . '/vendor/qcubed/plugin');
-			define ('__IMAGE_CACHE__', __APP_IMAGE_ASSETS__ . '/cache');
+			define ('__IMAGE_CACHE__', __IMAGE_ASSETS__ . '/cache');
```

Comment:
This too should be relative to the __TMP__ directory. __IMAGE_ASSETS__ is in the vendor directory, which is controlled by composer